Assets/MixedRealityToolkit.Examples/Demos/StandardShader/Scripts/MaterialParameterChanger.cs

```diff
@@ -0,0 +1,532 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Microsoft.MixedReality.Toolkit.UI;
+using UnityEngine.UI;
+using System;
+
+namespace Microsoft.MixedReality.Toolkit.Examples
+{
+    /// <summary>
+    /// This class changes the MRTKStandardShader parameter of the material attached using PinchSlider.
+    /// By acquiring each method in the event of PinchSlider.sc, you can see the function of MRTKStandardShader on the actual machine.
+    /// </summary>
+    public class MaterialParameterChanger : MonoBehaviour
+    {
+        /// <summary>
```

Comment:
Can you remove these comments per our coding guidelines?
https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html
"comments that restate what is obvious provide no extra information and increase the noise to signal ratio."

Reply:
Same with the items below

Reply:
I'll try.